tests/test_types.py

```diff
@@ -4481,7 +4481,7 @@ def test_bytesize_raises():
     class Model(BaseModel):
         size: ByteSize
 
-    with pytest.raises(ValidationError, match='should match'):
+    with pytest.raises(ValidationError, match='parse value'):
         Model(size='d1MB')
```

Comment:
```suggestion
    with pytest.raises(ValidationError, match='parse value') as exc_info:
        Model(size='d1MB')
    assert exc_info.value.errors(include_url=False) == [...]
```
easy way to make sure we know that we don't make this error message ugly again even if that short string is still present